prow/daily-release-qualification.sh

```diff
@@ -30,7 +30,12 @@ set -x
 source greenBuild.VERSION
 echo "Using artifacts from HUB=${HUB} TAG=${TAG} ISTIOCTL_URL=${ISTIOCTL_URL}"
 
-git clone https://github.com/istio/istio.git
+git clone -n https://github.com/istio/istio.git
+cd istio
+ISTIO_SHA=`curl $ISTIOCTL_URL/manifest.xml | grep istio/istio | cut -f 6 -d \"`
+git checkout $ISTIO_SHA
+cd ..
```

Comment:
I think we don't need to go up one level here. Just remove the `cd istio` on line 39 and we are good